src/docker/Dockerfile

```diff
@@ -63,42 +63,52 @@ RUN curl -LO https://github.com/derailed/k9s/releases/download/${K9S_VERSION}/k9
 # Install Helm 2
 ARG HELM2_VERSION=v2.16.1
 RUN curl -LO https://get.helm.sh/helm-${HELM2_VERSION}-linux-amd64.tar.gz && \
-    tar -zxvf helm-${HELM2_VERSION}-linux-amd64.tar.gz && \
-    mv linux-amd64/helm /usr/local/bin/helm2 && \
-    chmod +x /usr/local/bin/helm2 && \
-    rm -rf linux-amd64 helm-${HELM2_VERSION}-linux-amd64.tar.gz
+  tar -zxvf helm-${HELM2_VERSION}-linux-amd64.tar.gz && \
+  mv linux-amd64/helm /usr/local/bin/helm2 && \
+  chmod +x /usr/local/bin/helm2 && \
+  rm -rf linux-amd64 helm-${HELM2_VERSION}-linux-amd64.tar.gz
 
 # Install Helm 3
 ARG HELM3_VERSION=v3.0.2
 RUN curl -LO https://get.helm.sh/helm-${HELM3_VERSION}-linux-amd64.tar.gz && \
-    tar -zxvf helm-${HELM3_VERSION}-linux-amd64.tar.gz && \
-    mv linux-amd64/helm /usr/local/bin/helm && \
-    chmod +x /usr/local/bin/helm && \
-    rm -rf linux-amd64 helm-${HELM3_VERSION}-linux-amd64.tar.gz
+  tar -zxvf helm-${HELM3_VERSION}-linux-amd64.tar.gz && \
+  mv linux-amd64/helm /usr/local/bin/helm && \
+  chmod +x /usr/local/bin/helm && \
+  rm -rf linux-amd64 helm-${HELM3_VERSION}-linux-amd64.tar.gz
 
 # Install helmfile
 ARG HELMFILE_VERSION=v0.97.0
 RUN curl -sOL https://github.com/roboll/helmfile/releases/download/${HELMFILE_VERSION}/helmfile_linux_amd64 && \
-    chmod +x ./helmfile_linux_amd64 && \
-    mv helmfile_linux_amd64 /usr/local/bin/helmfile
+  chmod +x ./helmfile_linux_amd64 && \
+  mv helmfile_linux_amd64 /usr/local/bin/helmfile
 
 # Install Terraform
 ARG TERRAFORM_VERSION=0.12.19
 RUN curl -sOL https://releases.hashicorp.com/terraform/${TERRAFORM_VERSION}/terraform_${TERRAFORM_VERSION}_linux_amd64.zip && \
-    unzip terraform_${TERRAFORM_VERSION}_linux_amd64.zip -d /usr/local/bin/ && \
-    rm -f terraform_${TERRAFORM_VERSION}_linux_amd64.zip
+  unzip terraform_${TERRAFORM_VERSION}_linux_amd64.zip -d /usr/local/bin/ && \
+  rm -f terraform_${TERRAFORM_VERSION}_linux_amd64.zip
 
 # Install Azure cli
 ARG AZURE_CLI=2.0.78-1~buster
 RUN curl -sL https://packages.microsoft.com/keys/microsoft.asc | \
-    gpg --dearmor > /etc/apt/trusted.gpg.d/microsoft.asc.gpg && \
-    echo "deb [arch=amd64] https://packages.microsoft.com/repos/azure-cli/ buster main" | \
-    tee -a /etc/apt/sources.list.d/azure-cli.list && \
-    install_packages azure-cli=${AZURE_CLI}
+  gpg --dearmor > /etc/apt/trusted.gpg.d/microsoft.asc.gpg && \
+  echo "deb [arch=amd64] https://packages.microsoft.com/repos/azure-cli/ buster main" | \
+  tee -a /etc/apt/sources.list.d/azure-cli.list && \
+  install_packages azure-cli=${AZURE_CLI}
 
 # Quit being root for safety.
 USER dadsgarage
 
+# Install Node Version Manager, NodeJS, and NPM
+ARG NVM_VERSION=0.35.2
+ARG NODE_VERSION=node
```

Comment:
which node version gets installed can we make this more explicit?

Reply:
`node` is an alias NVM uses for the latest version. I can change it to pin the version.

Reply:
Considering that all other applications are pinned at a specific version I think the same should apply here and the default should be the current LTS with 12.15 being the current LTS version.